### DISCUSSION TITLE
为什么中间节点的响应时间比最终节点响应时间还要长？

### DISCUSSION THREAD

**Discussion by 5UFKEFU** [nxtrace/NTrace-core#250] [General]
```
sudo nexttrace xxxxx
NextTrace 1.3.1 2024-05-30T04:26:46Z brew
[NextTrace API] preferred API IP - 172.67.155.192 - 55.67ms - Misaka.HKG
IP Geo Data Provider: LeoMoeAPI
traceroute to 116.211.58.xxx  (u), 30 hops max, 52 bytes payload
1   192.168.31.1    *                         RFC1918          
                                              2.88 ms / 4.11 ms / 2.85 ms
2   *
3   10.96.20.150    *                         RFC1918          
                                              4.69 ms / 8.44 ms / 4.14 ms
4   47.246.60.146   AS24429                   中国 香港   alibabacloud.com 
                                              5.46 ms / 5.49 ms / 5.23 ms
5   47.246.60.145   AS24429                   中国 香港   alibabacloud.com 
                                              8.05 ms / 14.56 ms / 8.30 ms
6   203.12.205.133  AS4809   [CTHKI-HK]       中国 香港   www.chinatelecom.com.cn  电信
                                              6.96 ms / 6.11 ms / 5.70 ms
7   59.43.184.166   *        [CN2-BackBone]   中国 北京 北京  chinatelecom.cn  电信
                                              7.91 ms / 5.18 ms / 4.94 ms
8   59.43.183.109   *        [CN2-BackBone]   中国 广东 广州 X-I chinatelecom.cn  电信
                                              11.38 ms / 33.33 ms / 14.59 ms
9   59.43.187.109   *        [CN2-BackBone]   中国 广东 广州  chinatelecom.cn  电信
                                              * ms / * ms / 70.10 ms
10  59.43.130.149   *        [CN2-BackBone]   中国 广东 广州  chinatelecom.cn  电信
                                              12.26 ms / 13.45 ms / 14.04 ms
11  59.43.137.234   *        [CN2-BackBone]   中国 江西 南昌  chinatelecom.cn  电信
                                              29.97 ms / 36.55 ms / 34.13 ms
12  59.43.68.6      *        [CN2-BackBone]   中国 湖北 武汉  chinatelecom.cn  电信
                                              * ms / 197.35 ms / 186.35 ms
13  119.97.254.102  AS4134   [CHINANET-HB]    中国 湖北 武汉  www.chinatelecom.com.cn 
                                              71.88 ms / 76.52 ms / 71.77 ms
14  119.97.144.250  AS4134   [CHINANET-HB]    中国 湖北 武汉  www.chinatelecom.com.cn 
                                              76.72 ms / 71.14 ms / 70.85 ms
15  116.211.58.2  AS4809   [CHINANET-HB]    中国 湖北 武汉  www.chinatelecom.com.cn 
                                              32.35 ms / 36.14 ms / 34.56 ms
```

比如 你们看到 13，14，和 15，响应时间 反而小了？

**Comment by haima3** [comment]
这种情况其实比较常见，可能的原因也有很多种，例如QoS优先级不同 途径的各路由器负载情况 时间段 等因素都有可能影响

________________________________
From: Sarah Divento ***@***.***>
Sent: Saturday, July 20, 2024 2:13:11 AM
To: nxtrace/NTrace-core ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [NextTrace Community ] 为什么中间节点的响应时间比最终节点响应时间还要长？ (Discussion #250)


sudo nexttrace xxxxx
NextTrace 1.3.1 2024-05-30T04:26:46Z brew
[NextTrace API] preferred API IP - 172.67.155.192 - 55.67ms - Misaka.HKG
IP Geo Data Provider: LeoMoeAPI
traceroute to 116.211.58.xxx  (u), 30 hops max, 52 bytes payload
1   192.168.31.1    *                         RFC1918
                                              2.88 ms / 4.11 ms / 2.85 ms
2   *
3   10.96.20.150    *                         RFC1918
                                              4.69 ms / 8.44 ms / 4.14 ms
4   47.246.60.146   AS24429                   中国 香港   alibabacloud.com
                                              5.46 ms / 5.49 ms / 5.23 ms
5   47.246.60.145   AS24429                   中国 香港   alibabacloud.com
                                              8.05 ms / 14.56 ms / 8.30 ms
6   203.12.205.133  AS4809   [CTHKI-HK]       中国 香港   www.chinatelecom.com.cn  电信
                                              6.96 ms / 6.11 ms / 5.70 ms
7   59.43.184.166   *        [CN2-BackBone]   中国 北京 北京  chinatelecom.cn  电信
                                              7.91 ms / 5.18 ms / 4.94 ms
8   59.43.183.109   *        [CN2-BackBone]   中国 广东 广州 X-I chinatelecom.cn  电信
                                              11.38 ms / 33.33 ms / 14.59 ms
9   59.43.187.109   *        [CN2-BackBone]   中国 广东 广州  chinatelecom.cn  电信
                                              * ms / * ms / 70.10 ms
10  59.43.130.149   *        [CN2-BackBone]   中国 广东 广州  chinatelecom.cn  电信
                                              12.26 ms / 13.45 ms / 14.04 ms
11  59.43.137.234   *        [CN2-BackBone]   中国 江西 南昌  chinatelecom.cn  电信
                                              29.97 ms / 36.55 ms / 34.13 ms
12  59.43.68.6      *        [CN2-BackBone]   中国 湖北 武汉  chinatelecom.cn  电信
                                              * ms / 197.35 ms / 186.35 ms
13  119.97.254.102  AS4134   [CHINANET-HB]    中国 湖北 武汉  www.chinatelecom.com.cn
                                              71.88 ms / 76.52 ms / 71.77 ms
14  119.97.144.250  AS4134   [CHINANET-HB]    中国 湖北 武汉  www.chinatelecom.com.cn
                                              76.72 ms / 71.14 ms / 70.85 ms
15  116.211.58.2  AS4809   [CHINANET-HB]    中国 湖北 武汉  www.chinatelecom.com.cn
                                              32.35 ms / 36.14 ms / 34.56 ms


比如 你们看到 13，14，和 15，响应时间 反而小了？

―

**Reply by 5UFKEFU** [reply]
基本上是可以排除不稳定的因素，应该是多个时间测试结果都是这样的。
我知道 最终结果是看最终的节点 IP 的时间，但从道理上讲不通啊，为什么途经的反而更慢呢？
